src/client/java/README.md

```diff
@@ -102,74 +96,64 @@ It supports both synchronous and asynchronous. The default is asynchronous. You
 
 ##### DAOS URIs
 
-DAOS FileSystem binds to schema, "daos". And DAOS URIs are in the format of "daos://\[authority\]//\[path\]".
-Both authority and path are optional. There are two types of DAOS URIs, with and without DAOS UNS path depending on
-where you want DAOS Filesystem get initialized and configured.
-
-* With DAOS UNS Path
-
-The simple form of URI is "daos:///\<your uns path\>\[/sub path\]". "\<your path\>" is your OS file path created
-with DAOS command or Java DAOS UNS method, DaosUns.create(). "\[sub path\]" is optional. You can create the UNS path
-with below command.
+* daos://<pool UUID>/<container UUID>/.
+* daos://<pool label>/<container label>/.
+* daos://[authority starts with "uns-id"]/<uns path>. The authority is optional.
+   If the authority doesn't start with "uns-id", it will be considered as either pool UUID or label. You will get
+unexpected parse error.
```

Comment:
Not sure why you need this uns-id thing. Isn't the uns able to resolve path or pool/container UUID/label by itself?

Reply:
> Not sure why you need this uns-id thing. Isn't the uns able to resolve path or pool/container UUID/label by itself?

It's due to the design of Hadoop Filesystem and our URI structure. Even we have our implementation of Hadoop interface, there is still some logic/code controlled by Hadoop impl, such as the cache of file system instance.  When we use URI1, "daos://pool1/cont1", to get instance of DAOS file system (dfs1) via Hadoop interface, Hadoop caches the instance and index it by schema ("daos") and authority ("pool1").  Be noted the "cont1" is not part of the index key. When you try to get new DAOS file system (dfs2) to another container, "cont2", under same pool, with URI2, "daos://pool1/cont2" in the same JVM process,  Hadoop returns you wrong DAOS file system, "dfs1" instead of "dfs2" since the schema and authority of the URI2 is same as the URI1. Hadoop just returns you the cached DAOS file system.

With other file system, like hdfs, the URI is something like, "hdfs://192.168.1.100:10080/". As you can see, IP + port is authority. With different hdfs URI, "hdfs://192.168.1.200:10080", we can get different HDFS file system automatically.

Fortunately, we have UNS which can take different authorities to distinguish different containers, like "daos://uns-id-1/unspath1" and "daos://uns-id-2/unspath2". "uns-id" prefix is to differentiate UNS authority from pool label.